app/build.gradle

```diff
@@ -100,6 +115,8 @@ dependencies {
 
     //Image downloading and Caching library
     implementation 'com.github.bumptech.glide:glide:4.14.2'
+    implementation 'androidx.activity:activity-compose:1.3.1'
+    debugImplementation "androidx.compose.ui:ui-test-manifest:$compose_version"
```

Comment:
Please move this to the tests section down

Reply:
Sorted